### DISCUSSION TITLE
Helm: deploying app role without configuring any application

### DISCUSSION THREAD

**Discussion by zen** [gravitational/teleport#9901] [Q&A] [answered]
Hi,

I'm trying to have clean config and manage all my applications dynamically using `app` API It looks however that when you are launching teleport with "App" role you need to provide at least one app configuration. Any idea how to avoid that?

**Comment by webvictim** [comment, marked as answer]
The Helm chart does currently require that you specify at least one app when starting in app mode - this is basically to prevent people from inadvertently failing to add an app and then wondering why it doesn't work.

We'd take a look at a PR to remove this necessity if you wanted to write one.
